### PR TITLE
some minor improvement

### PR DESCRIPTION
**Pull request by digi-booster**
changing wordy words, removing unnecessary wordy words and commas, and using some easy phase

> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)
some words are wordy and some are redundant or unnecessary.

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html [hunk 1]
```diff
@@ -8,17 +8,17 @@
 ---
 <div>{{CSSRef}}</div>
 
-<p>One of the most common style changes made to lists is a change in the indentation distance—that is, how far the list items are pushed over to the right. This often leads to frustration, because what works in one browser often doesn't have the same effect in another. For example, if you declare that lists have no left margin, they move over in Internet Explorer, but sit stubbornly in place in Gecko-based browsers.This article will help you understand the problems that can occur and how to avoid them.</span></p>
+<p>One of the most common style changes made to lists is a change in the indentation distance—that is, how far the list items are pushed over to the right. This often leads to frustration, because what works in one browser often doesn't have the same effect in another. For example, if you declare that lists have no left margin, they move over in Internet Explorer, but sit stubbornly in place in Gecko-based browsers. This article will help you understand the problems that can occur and how to avoid them.</span></p>
```
Comment (teoli2003) [inline files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:11]: I know it was there before, but there is an unmatched `</span>` at the end of the line. Could you remove it?

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html [hunk 2]
```diff
@@ -8,17 +8,17 @@
...
 
-<p>In order to understand why this is the case, and more importantly how to avoid the problem altogether, it's necessary to examine the details of list construction.</p>
+<p> To understand why this is the case, and more importantly how to avoid the problem altogether, it's necessary to examine the details of list construction.</p>
```
Comment (teoli2003) [inline files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:13]: nit: Leading space to remove.

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html [hunk 3]
```diff
@@ -44,15 +44,15 @@ <h2 id="Indenting_It_Twice">Indenting It Twice</h2>
...
+<p> To avoid this and get some indentation, there are only three options available to browser implementors.</p>
 
 <ol>
  <li>Give each <code>&lt;li&gt;</code> element a left margin.</li>
  <li>Give the <code>&lt;ul&gt;</code> element a left margin.</li>
  <li>Give the <code>&lt;ul&gt;</code> element some left padding.</li>
 </ol>
 
-<p>As it turns out, nobody seems to have used the first option. The second option was taken by Internet Explorer for Windows and Macintosh, and Opera. The third was adopted by Gecko, and by extension all the browsers that embed it.</p>
+<p>As it turns out, nobody seems to have used the first option. The second option was taken by Internet Explorer for Windows and Macintosh, and Opera. The third was adopted by Gecko and by extension all the browsers that embed it.</p>
```
Comment (teoli2003) [inline files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:55]:
```suggestion
<p>As it turns out, nobody seems to have used the first option. The second option was taken by Internet Explorer, and Opera. The third was adopted by Firefox.</p>
```
Embedded Geckos and IE for Mac are long dead. Let's simplify a bit.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Almost good to go.

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html (hunk 1 above)
Comment (hv48067) [reply files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:11]: No, I didn't remove it, it wasn't already.

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html (hunk 3 above)
Comment (hv48067) [reply files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:55]: Oh, I don't know about thanks.

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:11]: I know, but can you fix it as we edit this page :-)

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html [hunk 4]
```diff
@@ -74,15 +74,15 @@ <h2 id="Finding_Consistency">Finding Consistency</h2>
...
 
 <pre class="brush:css">ul {margin-left: 0; padding-left: 0;}</pre>
 
 <p>Remember, though, that in so doing, you'll have the bullets hanging outside the list and its parent element. If the parent is the <code>body</code>, there's a strong chance your bullets will be completely outside the browser window, and thus will not be visible.</p>
 
 <h2 id="Conclusion">Conclusion</h2>
 
-<p>In the end, we can see that none of the browsers mentioned in this article is right or wrong about how they lay out lists. They use different default styles, and that's where the problems creep in. By making sure you style both the left padding and left margin of lists, you can find much greater cross-browser consistency in your list indentation.</p>
+<p>In the end, we can see that none of the browsers mentioned in this article is right or wrong about how they layout lists. They use different default styles, and that's where the problems creep in. By making sure you style both the left padding and left margin of lists, you can find much greater cross-browser consistency in your list indentation.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:85]: The change to this paragraph should be reverted. *layout* is a noun or adjective. *lay out* is a verb. See https://www.grammar.com/layout_vs._lay_out and other style guides. (And note this is similar to *markup* vs *mark up*.)

**Comment by teoli2003** [comment]
Feel free to re-request my review, once you have committed the fixes.

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html (hunk 4 above)
Comment (hv48067) [reply files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:85]: The layout is a noun that means the way something is arranged.
Lay out is a verb phrase that refers to the act of arranging something.
ok got it, thanks

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html]: It is not matched by an opening `<span>`, making the URL invalid. It is a remnant of a removed `<span class="seoSummary>` that we remove.

File: files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html (hunk 4 above)
Comment (teoli2003) [reply files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html:85]: You still need to revert this change.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
You still need to revert layout -> lay out

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Congratulations upon your first merged commit here!

**Comment by hv48067** [comment]
> Thanks a lot!
> 
> Congratulations upon your first merged commit here!

Thank you very much sir for your wonderful guidance and for helping me as a beginner and for patiently correcting my mistake.

**Pull request merged**